src/server/GA_parameters.py

```diff
@@ -0,0 +1,25 @@
+class GA_parameters:
```

Comment:
Please add a class doc string. Python's class names should be written using UpperCamelCase